crates/std_detect/src/detect/cache.rs

```diff
@@ -194,8 +171,18 @@ pub(crate) fn test<F>(bit: u32, f: F) -> bool
 where
     F: FnOnce() -> Initializer,
 {
-    if CACHE.is_uninitialized() {
-        initialize(f());
+    #[cfg(target_pointer_width = "32")]
```

Comment:
I think this `#[cfg]` could be avoided and the math here could be done with `mem::size_of::<usize>()` as well